examples/python/phi3v.py

```diff
@@ -83,5 +107,14 @@ def run(args: argparse.Namespace):
     parser.add_argument(
         "-p", "--provider", type=str, required=True, help="Provider to run model"
     )
+    parser.add_argument(
+        "--image_paths", nargs='*', type=str, required=False, help="Path to the images"
+    )
+    parser.add_argument(
+        '-pr', '--prompt', required=False, help='Input prompts to generate tokens from.'
+    )
+    parser.add_argument(
+        '--interactive', default=False, required=False, help='Interactive mode'
```

Comment:
Can we keep the existing behavior as the default? Users will be referring to this script when running the new vision models and our published examples on [the ORT website](https://onnxruntime.ai/docs/genai/tutorials/phi3-v.html) and on [GitHub](https://github.com/microsoft/onnxruntime-genai/blob/main/examples/python/phi-3.5-vision.md#4-run-phi-35-vision-onnx-models) refer to `-m` and `-p` as the only required flags.

Reply:
Yes, that's fine. Since the extra arguments are for CI testing, we should note that information in the `help` section for each of the new arguments. Otherwise, users may try to use the new arguments when seeing which arguments are available with `python phi3v.py --help` when the new arguments don't need to be used for the default behavior.

This comment is mainly referring to the current usage of `--interactive` as it changes the script's default behavior.

https://github.com/microsoft/onnxruntime-genai/blob/d9db677ce5ac4d8d23257c1402ada1b581609913/examples/python/phi3v.py#L40-L58

https://github.com/microsoft/onnxruntime-genai/blob/d9db677ce5ac4d8d23257c1402ada1b581609913/examples/python/phi3v.py#L75-L81

Since the `if` cases are what's originally in the script, the script's default behavior will be non-interactive and will require `--interactive` to be passed for interactive mode. We should make interactive mode the default for users and use non-interactive mode with a flag for CI testing.

Reply:
Got it. I'll reverse the default of `interactive` to `True` and change the CI script accordingly